jerry-core/ecma/operations/ecma-objects-arguments.c

```diff
@@ -99,84 +95,80 @@ ecma_op_create_arguments_object (ecma_object_t *func_obj_p, /**< callee function
 
   ecma_property_descriptor_t prop_desc = ecma_make_empty_property_descriptor ();
 
-  if (bytecode_data_p != NULL)
```

Comment:
This check is removed because it is unnecessary. The result is a lot of code realignment, so the changes below are mostly caused by that.